src/python/WMComponent/DBS3Buffer/MySQL/DBSBufferFiles/AddChecksumByLFN.py

```diff
@@ -21,9 +21,10 @@ def execute(self, lfn = None, cktype = None, cksum = None, bulkList = None, conn
         if bulkList:
             binds = bulkList
         else:
-            binds = {'lfn': lfn, 'cktype': cktype, 'cksum': cksum}
+            binds = [{'lfn': lfn, 'cktype': cktype, 'cksum': cksum}]
 
-        result = self.dbi.processData(self.sql, binds,
-                         conn = conn, transaction = transaction)
+        for sliceBinds in grouper(binds, 10000):
```

Comment:
same for this one as well. we slice it in processData. If this give the performance boost. That means we need to change to the code in processData. (Maybe using loop instead of recursive call. - Although I don't think that makes big performance difference)